sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultAccessControlClient.cs

```diff
@@ -203,7 +203,7 @@ public virtual async Task<Response<KeyVaultRoleDefinition>> GetRoleDefinitionAsy
         /// <param name="roleDefinitionName">The unique role definition name. If null a unique role definition name will be generated.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
-        [CallerShouldAudit(Reason = KeyVaultAdministrationClientOptions.CallerShouldAuditReason)]
+        [CallerShouldAudit(reason: KeyVaultAdministrationClientOptions.CallerShouldAuditReason)]
```

Comment:
Nit: this works (obviously it compiles) but is atypical. Can we just remove the `reason: ` named parameter in all instances?